lib/libutee/tee_api_operations.c

```diff
@@ -444,18 +451,24 @@ TEE_Result TEE_SetOperationKey2(TEE_OperationHandle operation,
 	TEE_Result res;
 	uint32_t key_size = 0;
 
-	if (operation == TEE_HANDLE_NULL)
-		TEE_Panic(0);
+	if (operation == TEE_HANDLE_NULL) {
+		res = TEE_ERROR_BAD_PARAMETERS;
+		goto out;
+	}
 
 	/* Two keys not expected */
```

Comment:
I can see that this hasn't been changed by you, but is this really correct? Reading the spec it says:
```
TEE_HANDLE_FLAG_EXPECT_TWO_KEYS : Set if the algorithm expects two keys to be set, using
TEE_SetOperationKey2 . This happens only if algorithm is set to TEE_ALG_AES_XTS .
```

And in `TEE_SetOperationKey2` one can read:
```
The TEE_SetOperationKey2 function initializes an existing operation with two keys. This is used only for
the algorithm TEE_ALG_AES_XTS .
```